cli.py

```diff
@@ -1,107 +0,0 @@
-"""
```

Comment:
I moved this with `git mv cli.py cli/cli.py`, but for whatever reason it's showing as a delete + new file.